client/landing/gutenboarding/onboarding-block/question/index.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+interface QuestionProps {
+	label: string;
+	displayValue: string;
+	children: React.ReactNode;
+	isActive: boolean;
+	onExpand: () => void;
+}
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+const Question = ( { isActive, onExpand, displayValue, label, children }: QuestionProps ) => (
+	<div
+		className={ classNames( {
+			'onboarding-block__question': true,
+			selected: isActive,
+		} ) }
+	>
+		<span>{ label }</span>
+		<div className="">
+			{ isActive ? (
+				children
+			) : (
+				<>
+					<button className="onboarding-block__question-answered" onClick={ onExpand }>
+						{ displayValue }
+					</button>
+					<span>.</span>
+				</>
+			) }
+		</div>
+	</div>
+);
+
+export default Question;
```

Comment:
I'm not entirely sure if the `Question` component gives us that much value, or if we should just merge the logic into each individual step component. After all, it's basically just a ternary, isn't it? :thinking:

Reply:
This component is handling the structure of each question (wrapper, static label, link to expand the question) and it's also in charge with generic styling. By extracting this stuff here we not only save some code duplication but also guarantee consistency and flexibility. I think this is what composition is about, right?